src/components/views/Booktrak/BooktrakListings.tsx

```diff
@@ -4,98 +4,158 @@ import React, { useState, useEffect } from "react";
 // Redux/ Routing imports
 import { useAppSelector } from "../../../lib/store";
 import { getWSO } from "../../../lib/authSlice";
-import { Link, useNavigate } from "react-router-dom";
+import { useNavigate } from "react-router-dom";
 
 // Additional Imports
 import {
   ModelsBook,
   ModelsBookListing,
 } from "wso-api-client/lib/services/types";
-import { BookConditionEnumToString } from "./BooktrakUtils";
+import PaginationButtons from "../../PaginationButtons";
+import BooktrakListingsTable from "./BooktrakListingsTable";
+import "../../stylesheets/Booktrak.css";
+import { ModelsBookListingWithTitle } from "./BooktrakUtils";
 
-const BooktrakListings = ({ book }: { book: ModelsBook }) => {
+type ServerError = {
+  errorCode: number;
+  message: string;
+};
+
+function isServerError(error: unknown): error is ServerError {
```

Comment:
Move this to `lib/general.js`. Also rewrite it as `export const isServerError` instead of function.